src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -46,9 +47,14 @@ protected function configure()
 
 The <info>Environment</info> section displays the current environment variables managed by Symfony Dotenv. It will not
 be shown if no variables were found. The values might be different between web and CLI.
+
+Passing <info>eolCheck</info> as an option you will get an error if the current symfony kernel is End of Maintenance and
+End of Life (can be used in CI as check `php bin/console about --eolCheck`
+
 EOT
-            )
-        ;
+            )->addOption(
+        'eolCheck'
+            );
```

Comment:
Options should be defined before the description, on one line, and fully defined.